app/views/timeline_entries/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name} timeline entries",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Add timeline entry"
+    },
+  ]
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, formatted_title(@coronavirus_page) %>
+<% content_for :context, "Add timeline entry" %>
+<% if @timeline_entry.errors.any? %>
+  <%= render "shared/sub_sections/form_errors", resource: @timeline_entry %>
+<% end %>
+
+<div class="covid-edit-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @timeline_entry,
```

Comment:
I don't think this model does anything - since we reference the [model directly in the form](https://github.com/alphagov/collections-publisher/pull/1235/files#diff-0ed636e7df998d6718cbb06b6a880f21c098ce3020e9bed952165b9018b01794R21). So I think we can remove this without consequences.

Reply:
Ah, yes, I think this was one of the many options I tried to get the form to submit before stumbling on the `local` issue. I'll remove it.